test/iron-overlay-behavior.html

```diff
@@ -155,6 +155,21 @@
           assert.equal(getComputedStyle(overlay).display, 'none', 'overlay starts hidden');
         });
 
+        test('_renderOpened called only after is attached', function(done) {
+          var overlay = document.createElement('test-overlay');
+          // overlay is ready at this point, but not yet attached.
+          var spy = sinon.spy(overlay, '_renderOpened');
+          // this triggers _renderOpened
```

Comment:
nit: Capitalize, here and below, since you do that in the middle (and probably everywhere else in the file)